### PR TITLE
feat: add `atlas pull`

### PR DESCRIPTION
**Pull request by OmarIthawi**
Atlas pull in the most simple form. Depends on https://github.com/overhangio/tutor/pull/964. 

-----------------

This pull request is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which implements the [Translation Infrastructure update OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html).

### CODE REVIEW

**Comment by OmarIthawi** [comment]
cc: @regisb

File: tutorcredentials/templates/credentials/build/credentials/Dockerfile [hunk 1]
```diff
@@ -125,6 +125,9 @@ ENV PATH /openedx/venv/bin:./node_modules/.bin:/openedx/nodeenv/bin:${PATH}
 ENV VIRTUAL_ENV /openedx/venv/
 WORKDIR /openedx/credentials
 
+RUN atlas pull --repository="{{ ATLAS_REPOSITORY }}" --branch="{{ ATLAS_BRANCH }}" translations/credentials/credentials/conf/locale:credentials/conf/locale
```
Comment (OmarIthawi) [inline tutorcredentials/templates/credentials/build/credentials/Dockerfile:128]:
```suggestion
RUN atlas pull --repository="{{ ATLAS_REPOSITORY }}" --branch="{{ ATLAS_BRANCH }}" {{ ATLAS_OPTIONS }} translations/credentials/credentials/conf/locale:credentials/conf/locale
```

**Comment by OmarIthawi** [comment]
@Faraz32123 this is ready for review.

I've tested it on my devstack and the `atlas pull` seems to be working fine.

Please let me know if there's anything I need to check.

**Review by Faraz32123 — COMMENTED** [review]
LGTM! Thanks for your contribution @OmarIthawi.
Forgot to submit review before merge.

**Pull request merged**